hvillacrusisCT/dotTheory---Javascript
# JS-690: Build a Quiz Challenge

## Related Ticket

[JS-690](https://codeandtheory.atlassian.net/browse/JS-690)

## Description
Added the Quiz functionality to the application, allowing users to take a quiz and get their results.

## Changes
- Created a `Quiz` class in `quiz.js`.
- Class includes methods for asking questions, comparing user responses to correct answers, and displaying results.
- Added a multidimensional array to store quiz questions and their corresponding answers.
- The number of correctly answered questions is stored and displayed to the user after all questions have been answered.

## Validation
1. Make sure the `quiz.js` file is in the same directory as the `index.html`.
2. Open the `quiz.js` file.
3. The `Quiz` class should be present.
4. Inside the class, the `questions` array should store question-answer pairs.
5. The `correctCount` variable should store the number of correctly answered questions.
6. The `askQuestions`, `displayResults`, and `runQuiz` methods should be present.
7. Open the `index.html` file in a web browser. This should automatically run the quiz from the `quiz.js` file.
8. Interact with the quiz through the browser's prompt and alert dialogs to test the functionality.
